DeAceroDLabs/marketplace-frontend
# [DMP-524] Add Address Section to Payment View

## What is done

[[DMP-524] Sección de dirección](https://deacero.atlassian.net/browse/DMP-524)

## How is done

1. Added a new section (Address) to the Payment view.
2. Added the necessary config to call to the get user by email endpoint.
3. Modified the User Context to receive the User value as the backend service sends it.
4. Added the user context to the app, it uses a mock value while the new endpoint is done.

## How to test

1. Run the app `npm start`.
5. Go to `/carrito/pago`.
6. Check how you will see the new "Direccion" section added.
